testing/src/main/java/io/opencensus/testing/propagation/HttpUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.testing.propagation;
+
+/**
+ * Utilities for HTTP testing.
+ *
+ * @since 0.13
+ */
+public final class HttpUtils {
```

Comment:
Is this class also meant to be used by users of OpenCensus?  If not, I think it would be better to put it in a separate library.  I think opencensus-testing is meant to provide utilities for users of OpenCensus to test their use of the API, such as fake implementations.

/cc @bogdandrutu

Reply:
I don't think it will be used by users. But it would be used by impl_core and contrib_http_util. Since it is test related, I put it in opencensus-testing, not opencensus-api. Not sure if there are better places.